Tests/Unit/Document/DocumentTests.swift

```diff
@@ -789,14 +787,36 @@ class DocumentTests: XCTestCase {
         }
     }
 
+    // swiftlint: disable force_cast
     func test_change_paths_test_for_array() async throws {
```

Comment:
It would be better to include a test for the unsubscribe.

https://github.com/yorkie-team/yorkie-js-sdk/blob/20d44ac2eb31f80863cdab938b86657d7645a389/test/integration/document_test.ts#L449-L464

Reply:
That comment only disables linting codes. The test will be performed.

Reply:
I apologize for the confusion. I misplaced the comment. Regardless of its placement, I wanted to express that having a test for the unsubscribe would be good.

Reply:
I added the unsubscribe() function and related Test code. Please review it.

@chacha912

Reply:
Thank you for adding the feature! I have a few questions about the differences from yorkie-js-sdk.
- Is it possible to cancel the defaultSubscribeCallback?
- Can we register multiple callbacks for a single target?

Reply:
> Thank you for adding the feature! I have a few questions about the differences from yorkie-js-sdk.
> 
> * Is it possible to cancel the defaultSubscribeCallback?
Yes. Call unsubscribe() without parameter.
> * Can we register multiple callbacks for a single target?
No. Only one callback could be registered.